include/nw4r/g3d/g3d_resmat.h

```diff
@@ -123,6 +123,16 @@ struct ResGenMode : public ResCommon<ResGenModeData> {
     void GXSetCullMode(GXCullMode);
 };
 
+struct ResTexObjData {
+};
+
+struct ResTexObj {
+    ResCommon<ResTexObjData> mTexObj;
+    inline ResTexObj(void *vptr) : mTexObj(vptr) {}
```

Comment:
For Future notes (will fix in future g3d pass):
 nw4r objects starting with `Res` will inherit usually from ResCommon in the following fashion:
 `class Res[NAME] : public ResCommon<Res[NAME]Data> { }`
 The Data isn't always the case, but often times is